src/Calculator.ts

```diff
@@ -155,6 +159,14 @@ export function calculateMeasureReports(
     // create group population counts from result's detailedResults (yes/no->1/0)
     report.group = [];
     result?.detailedResults?.forEach(dr => {
+      // add narrative for relevant clauses
+      if (dr.html) {
+        report.text = {
```

Comment:
For measures with multiple population groups this would only have the HTML for the last population group on the report. They should be appended together.

Reply:
Good catch. I just pushed up code that appends rather than overwrites.

To test this, I actually modified the exm130 bundle I was using. I just added a second group in the measure resource within that bundle which is identical to group-1. The resulting HTML narrative had both groups together.